dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -226,6 +226,11 @@ Block Aggregator::getHeader(bool final) const
     return params.getHeader(final);
 }
 
+Block Aggregator::getSourceHeader() const
+{
+    return params.src_header;
+}
```

Comment:
why need this？
how about adding some comments here?

Reply:
Done